node-agent/node_manager/Cargo.toml

```diff
@@ -5,4 +5,9 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+name = "node_manager"
+path = "src/lib.rs"
+
```

Comment:
I think that's probably not needed.

Reply:
Yes, you are rigth, It's old stuff, I will remove that.

Thank you.